opm/io/eclipse/ESmry.hpp

```diff
@@ -24,12 +24,16 @@
 #include <string>
 #include <unordered_map>
 #include <vector>
+#include <map>
```

Comment:
You should also
```C++
#include <stdint.h>
```
here since we're now exposing interfaces that are defined in terms of `int64_t`.